src/api/common_api.js

```diff
@@ -16,6 +16,13 @@ module.exports = {
 
     definitions: {
 
+
+        multipart: {
```

Comment:
This identifier is too short and hard to understand what I will find inside.
I would rename to `multipart_num`.